services/org/org.go

```diff
@@ -39,6 +40,10 @@ func DeleteOrganization(org *organization.Organization) error {
 		return models.ErrUserOwnPackages{UID: org.ID}
 	}
 
+	if err := secret_service.DeleteSecretsByOwnerID(ctx, org.ID); err != nil {
```

Comment:
I thought you would simply use `models/DeleteUser#72-91`.

Reply:
Solved in the new PR.